src/master-playlist-controller.js

```diff
@@ -272,17 +274,22 @@ export class MasterPlaylistController extends videojs.EventTarget {
       loaderType: 'audio'
     }));
 
+    segmentLoaderOptions.loaderType = 'vtt';
+    this.subtitleSegmentLoader_ = new VTTSegmentLoader(segmentLoaderOptions);
+
     this.decrypter_.onmessage = (event) => {
       if (event.data.source === 'main') {
         this.mainSegmentLoader_.handleDecrypted_(event.data);
       } else if (event.data.source === 'audio') {
         this.audioSegmentLoader_.handleDecrypted_(event.data);
+      } else if (event.data.source === 'vtt') {
+        this.subtitleSegmentLoader_.handleDecrypted_(event.data);
       }
     };
 
     this.setupSegmentLoaderListeners_();
 
-    this.masterPlaylistLoader_.start();
+    this.masterPlaylistLoader_.load();
```

Comment:
@gesinger and I noticed that `load` calls `start` if start hasn't been called yet and is safe to call multiple times, but `start` should only be called once. Since this call is in the `MasterPlaylistController` constructor, theres no risk of running into that scenario, but we decided to just change any calls to `start` into `load` so we didn't have to concern ourselves with it. We can change it back if you think `start` makes more sense semantically